packages/aws-cdk/lib/init-templates/v1/app/typescript/cdk.template.json

```diff
@@ -1,3 +1,18 @@
 {
-  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts"
+  "app": "npx ts-node --prefer-ts-exts bin/%name%.ts",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "jest.config.js",
```

Comment:
I _think_ this is covered by the above JS file exclusion already, no?

Same comment in the other TS templates.